.travis.yml

```diff
@@ -1,17 +1,12 @@
 language: node_js
 node_js:
   - "8"
+before_install: 
+  - chmod +x web_deploy.sh
```

Comment:
Why not to commit the executable flag in the repository?

Reply:
5ebed79